File: bds/payment_service/src/main/java/com/web/bds/entity/PaymentHistory.java
package com.web.bds.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.CreationTimestamp;

import java.util.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaymentHistory {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id_paymenthistory;

    @Column
    private int id_payment;

    @Column
    private double pre_amount;

    @Column
    private double pay_money;

    @Column
    private double aft_amount;

    @Column
    private int person_modified;

    @Column
    @CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)
    private Date date_modified;
}

File: frontend/src/components/admin/Product/PaymentHistory.js
import React from 'react'


const PaymentHistory = ({ payment }) => {



  return (

    <tr key={payment.id_paymenthistory}>
      <td className="text-left px-6 py-4 whitespace-nowrap">{payment.id_paymenthistory}</td>

      <td className="text-left px-6 py-4 whitespace-nowrap">{payment.pre_amount}</td>

      <td className="text-left px-6 py-4 whitespace-nowrap">{payment.pay_money}</td>
     
      <td className="text-left px-6 py-4 whitespace-nowrap">{payment.aft_amount}</td>

      <td className="text-left px-6 py-4 whitespace-nowrap">{payment.date_modified}</td>
    </tr>
  )
}

export default PaymentHistory;